src/MongoDB.Driver.Core/Core/Operations/RetryabilityHelper.cs

```diff
@@ -109,6 +109,10 @@ public static bool IsResumableChangeStreamException(Exception exception, Semanti
             {
                 return true;
             }
+            if (exception is MongoCommandException mongoCommandException && (ServerErrorCode)mongoCommandException.Code == ServerErrorCode.CursorNotFound)
```

Comment:
we have unit tests for this logic

Reply:
We do?

Reply:
here: https://github.com/MikalaiMazurenka/mongo-csharp-driver/blob/master/tests/MongoDB.Driver.Core.Tests/Core/Operations/RetryabilityHelperTests.cs
Previously we also had this test: https://github.com/DmitryLukyanov/mongo-csharp-driver/commit/ea4d8d3e5ff7b7cb6d411ce36096b53c3e358373#diff-02ff3937bce6086193697cd0939acce6L130

Reply:
Added unit tests.
Done.